src/main/java/com/zendesk/maxwell/producer/RabbitmqProducer.java

```diff
@@ -0,0 +1,57 @@
+package com.zendesk.maxwell.producer;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Created by hitesh on 30/12/16.
+ */
+public class RabbitmqProducer extends AbstractProducer {
+
+    static final Logger LOGGER = LoggerFactory.getLogger(RabbitmqProducer.class);
+    private static String exchangeName;
+    private Connection connection;
+    private Channel channel;
+    public RabbitmqProducer(MaxwellContext context) {
+        super(context);
+        this.exchangeName = context.getConfig().rabbitmqExchange;
+
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setHost(context.getConfig().rabbitmqHost);
+        try {
+            this.connection = factory.newConnection();
+            this.channel = connection.createChannel();
+            this.channel.exchangeDeclare(exchangeName, context.getConfig().rabbitmqExchangeType);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (TimeoutException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void push(RowMap r) throws Exception {
+        String value = r.toJSON(outputConfig);
+        if (value == null) {
+            return;
+        }
+        String routingKey = r.getDatabase() + "." + r.getTable();
+
+        channel.basicPublish(exchangeName, routingKey, null, value.getBytes());
```

Comment:
Can you suggest what other alternatives could be? I had hard time in thinking about alternate, and hence skipped it earlier. 

Would adding support for static routing key be of any use?
Would adding support till column level help some possible usecases?